src/test/java/com/losthex/AppTest.java

```diff
@@ -0,0 +1,12 @@
+package com.losthex;
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+public class AppTest {
+
+    @Test
+    public void testApp() {
+        assertTrue(true);
```

Comment:
add a test that will check arguments length (if not 4 or 5 - assert that it throws the correct exception type)

and another test that throws error for random arguments